portfolio/src/main/webapp/Anime.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <header>
+          <h1>Anime</h1>
+      </header>
+      <nav>
+          <a href="index.html">Home</a>
+          <a href="Anime.html">Anime</a>
+          <a href="family.html">Family</a>   
+      </nav>
+      <div>
+          <h1>List Of My Favourites</h1>
+          <ul>
+              <li>
+                  <h3>Ouran Highschool Host Club</h3>
+                  <p>
+                      Throwback to middleschool weekends with my siblings<br>
+                      This anime was hilarious and great for unwinding and relaxing with the sibs
+                      </p>
+                  <img src="Anime/ouran.jpeg"/>
+              </li>
+              <li>
+                  <h3>Soul Eater</h3>
+                  <p>
+                      Also a throwback to middleschool weekends with my siblings<br>
+                      This anime was more serious but still great
+                  </p>
+                  <img src="Anime/souleater.jpg"/>
+              </li>
+              <li>
+                  <h3>Neo Yokio</h3>
+                  <p>
+                      Me and my siblings dicovered this spoof of an anime on Netflix<br>
+                      I will forever love the Toblerone because of this not for its taste but
+                      because it is forever iconic
+                  </p>
+                  <img src="Anime/neoyokio.png"/>
+              </li>
+              <li>
+                  <h3>Saga Of Tanya The Evil</h3>
+                  <p>
+                      This was a reccomendation from my brother he said I reminded him of Tanya...
+                      I cant say I agree with his assement.. but I cant say I dont disagree either
+                  </p>
+                  <img src="Anime/sagaof tanya.jpg"/>
```

Comment:
Avoid spaces in filepaths, it forces you to wrap paths in quotes when you're in the terminal. Use underscores or hyphens instead, just be consistent in using one or the other.

Reply:
which space?

Reply:
Space between `sagaof` and `tanya.jpg`

Reply:
oh I see i did that because there was a space in the image file so i renamed the file and fixed it